Hqub.MusicBrainz/Hqub.MusicBrainz.API/Cache/IRequestCache.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+namespace Hqub.MusicBrainz.API.Cache
+{
+    using System.IO;
+
+    /// <summary>
+    /// A simple cache interface.
+    /// </summary>
+    public interface IRequestCache
+    {
+        /// <summary>
+        /// Add a request and its response to the cache.
+        /// </summary>
+        void Add(string request, Stream response);
+
+        /// <summary>
+        /// Add a request and its response to the cache.
+        /// </summary>
+        /// <returns>True, if a cache entry matching the request was found.</returns>
+        bool TryGetCachedItem(string request, out Stream stream);
```

Comment:
Maybe this should return a `Task` to enable async calling. Same for `Add` method.